Source/common/santa.proto

```diff
@@ -948,6 +948,72 @@ message GatekeeperOverride {
   optional CodeSignature code_signature = 3;
 }
 
+// Information captured when Background Task Management (BTM) becomes aware
+// of a launch item being added or removed. This includes launch agents and
+// daemons as well as login items added by the user, via MDM or by an app.
+message LaunchItem {
+  // The process that emitted the event
+  optional ProcessInfoLight instigator = 1;
+
+  // Whether or not the launch item is being added or removed
+  enum Action {
+    ACTION_UNKNOWN = 0;
+    ACTION_ADD = 1;
+    ACTION_REMOVE = 2;
+  };
+
+  Action action = 2;
+
+  // The process that triggered the BTM operation. One will always be set.
+  // Note: Due to macOS system limitations, the process that triggered the
+  // event may have already exited before the event could be generated. This
+  // results in only a small subset of the information from the triggering
+  // process to be reported.
+  oneof btm_instigator {
+    ProcessInfoLight trigger_process = 3;
+    ProcessID trigger_id = 4;
+  }
+
+  // The app the registered the launch item. This field is optional and it is
+  // possible that neither case is set.
+  // Note: If the registering app has exited, only process id information will
+  // exist.
+  oneof app {
+    ProcessInfoLight registrant_process = 5;
+    ProcessID registrant_id = 6;
+  }
+
+  // The type of launch item corresponding to this event
+  enum ItemType {
+    ITEM_TYPE_UNKNOWN = 0;
+    ITEM_TYPE_USER_ITEM = 1;
+    ITEM_TYPE_APP = 2;
+    ITEM_TYPE_LOGIN_ITEM = 3;
+    ITEM_TYPE_AGENT = 4;
+    ITEM_TYPE_DAEMON = 5;
+  }
+
+  ItemType item_type = 7;
+
+  // Whether or not the launch item is a legacy plist
```

Comment:
Can we expand on what legacy means? Do we know?

Reply:
This seems to correlate with items backed by plists on disk vs SMAppService. But i'm not certain. Will leave largely undocumented for now since this is all the ES docs say.